modules/ROOT/pages/configure-teams.adoc

```diff
@@ -1,4 +1,4 @@
-= Configure Teams
+= Configuring Teams
```

Comment:
Maybe in the future, we can add more flair to these headings - like include the user value?